packages/org-unit-dialog/src/OrgUnitSelector.js

```diff
@@ -99,49 +101,70 @@ class OrgUnitSelector extends Component {
     };
 
     renderOptionsPanel = () => (
-        <Grid
-            spacing={8}
-            style={styles.footer.index}
-            container
-        >
-            <Grid item xs={4}>
-                <FormControl style={{ width: '100%' }}>
-                    <InputLabel htmlFor="level-select">{i18n.t('Level')}</InputLabel>
-                    <Select
-                        value={this.props.level}
-                        onChange={this.props.onLevelChange}
-                        input={<Input id="level-select" />}
-                        renderValue={this.renderLevelOptions}
-                        disabled={this.props.userOrgUnits.length > 0}
-                        multiple
-                        fullWidth
-                    >
-                        {this.props.levelOptions.map(option => (
-                            <MenuItem key={option.id} value={option.id}>{option.displayName}</MenuItem>
-                        ))}
-                    </Select>
-                </FormControl>
-            </Grid>
-            <Grid item xs={4}>
-                <FormControl style={{ width: '100%' }}>
-                    <InputLabel htmlFor="group">{i18n.t('Group')}</InputLabel>
-                    <Select
-                        value={this.props.group}
-                        onChange={this.props.onGroupChange}
-                        input={<Input name="group" id="group" />}
-                        renderValue={this.renderGroupOptions}
-                        disabled={this.props.userOrgUnits.length > 0}
-                        multiple
-                        displayEmpty
-                        fullWidth
-                    >
-                        {this.props.groupOptions.map(option => (
-                            <MenuItem key={option.id} value={option.id}>{option.displayName}</MenuItem>
-                        ))}
-                    </Select>
-                </FormControl>
+        <div style={styles.footer.index}>
+            <div style={styles.footer.tooltipContainer}>
```

Comment:
Since this tooltip isn't an option and its being absolutely positioned, it seems to be logically belong better to the div that wraps the OrgUnitTree. I was also thinking that "tooltip" is a bit misleading as a term. Maybe snackbar or something like that?